edge-modules/api-proxy-module/Cargo.toml

```diff
@@ -28,4 +28,4 @@ lazy_static = "1.4"
 matches = "0.1"
 mockito = "0.25"
 test-case = "1.0"
-tokio = { version = "0.2", features = ["macros"] }
\ No newline at end of file
+tokio = { version = "0.2", features = ["macros"] }
```

Comment:
revert it.

Reply:
No I actually need it . Otherwise I get failure telling me I need new line in toml.